Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldsHelper.cs

```diff
@@ -59,13 +59,13 @@ public int GetFieldCount(int artifactId, int workspaceId)
 			{
 				const string routeName = "GetFieldCountAsync";
 
-				var requestModel = new FieldCountBaseRequestModel
+			FieldCountBaseRequestModel requestModel = new FieldCountBaseRequestModel
```

Comment:
i like that you have updated them to actual types from var. will make code much easier to read.